### PR TITLE
Symfony 5 support

### PR DESCRIPTION
**Pull request by KDederichs**
Added Symfony 5 support by increasing the version boundary in composer and returning 0 in commands.
Also Fixed the vertex test and added ext-bcmath as requirement.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ composer.lock
 composer.phar
 phpunit.xml
 php-cs-fixer.phar
+.idea/
```
Comment (Nyholm) [inline .gitignore]: This should go in the global gitignore.

Comment (KDederichs) [reply .gitignore]: Ah sure, I'll just remove it and update the PR then.

**Comment by KDederichs** [comment]
Any chance you can make a new release with that?

**Comment by Nyholm** [comment]
Way ahead of you.

**Pull request merged**